packages/material-ui/src/Breadcrumbs/BreadcrumbCollapsed.test.js

```diff
@@ -13,35 +13,25 @@ describe('<BreadcrumbCollapsed />', () => {
   });
 
   it('should render an icon', () => {
-    const { container } = render(<BreadcrumbCollapsed />);
+    const { container, getByRole } = render(<BreadcrumbCollapsed />);
 
     expect(container.querySelectorAll('svg').length).to.equal(1);
-    expect(container.firstChild).to.have.class(classes.root);
+    expect(getByRole('button')).to.have.class(classes.button);
   });
 
-  describe('prop: onKeyDown', () => {
-    it(`should be called on key down - Enter`, () => {
-      const handleClick = spy();
-      const { container } = render(<BreadcrumbCollapsed onClick={handleClick} />);
-      const expand = container.firstChild;
-
-      act(() => {
-        expand.focus();
-        fireEvent.keyDown(expand, { key: 'Enter' });
-      });
+  it('renders a native <button>', () => {
```

Comment:
Added a separate test to make it explicit what is being tested instead of hiding it inside another test that is subject to possibly unrelated changes.